forge-gui/res/cardsfolder/t/telekinesis.txt

```diff
@@ -2,6 +2,7 @@ Name:Telekinesis
 ManaCost:U U
 Types:Instant
 A:SP$ Tap | Cost$ U U | ValidTgts$ Creature | SubAbility$ DBPump | SpellDescription$ Tap target creature. Prevent all combat damage that would be dealt by that creature this turn. It doesn't untap during its controller's next two untap steps.
```

Comment:
Probably cut `Prevent all combat damage that would be dealt by that creature this turn. It doesn't untap during its controller's next two untap steps.` from SpellDesc